code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -650,3 +650,23 @@
 	var/location = get_turf(holder.my_atom)
 	for(var/i in 1 to 10)
 		new /obj/item/stack/sheet/plastic(location)
+
+//Botany chemicals, ported from /VG/ #15432
+
+/datum/chemical_reaction/eznutrmient
+	name = "E-Z-Nutriment"
+	id = "eznutriment"
+	results = list("eznutriment" = 3)
+	required_reagents = list("nitrogen" = 1, "phosphorus" = 1, "potassium" = 1)
```

Comment:
@ExcessiveUseOfCobblestone what use would someone on lavaland have for these chems? Unless they were doing botany of course, in which case they would be able to use those plants to make these